### PR TITLE
ci: update size benchmarks for core with 9.0.0-next.8

### PR DESCRIPTION
**Pull request by kara**

### CODE REVIEW

**Review by AndrewKushnir — APPROVED** [review]

File: packages/core/test/bundling/core_all/bundle.golden_size_map_ivy.json [hunk 1]
```diff
@@ -177,48 +179,47 @@
...
+              "size": 5655,
+              "bindings.ts": 3235,
+              "map_based_bindings.ts": 941,
+              "state.ts": 403,
+              "styling_debug.ts": 1076
             },
             "tokens.ts": 10,
             "util/": {
-              "size": 3714,
+              "size": 6565,
```
Comment (kara) [inline packages/core/test/bundling/core_all/bundle.golden_size_map_ivy.json]: This diff doesn't make sense to me. It's about 3KB more, but the individual items in this folder are only increased by 500 bytes.

File: packages/core/test/bundling/core_all/bundle.golden_size_map_ivy.json [hunk 2]
```diff
@@ -131,38 +131,40 @@
...
               "ng_onchanges_feature.ts": 571,
               "providers_feature.ts": 113
             },
             "fields.ts": 187,
             "hooks.ts": 1748,
-            "i18n.ts": 8966,
+            "i18n.ts": 8980,
             "instructions/": {
-              "size": 21951,
+              "size": 24630,
```
Comment (kara) [inline packages/core/test/bundling/core_all/bundle.golden_size_map_ivy.json]: Note: mostly coming from styling code being moved out of the styling folder into here

File: packages/core/test/bundling/core_all/bundle.golden_size_map_ivy.json (hunk 1 above)
Comment (kara) [inline packages/core/test/bundling/core_all/bundle.golden_size_map_ivy.json]: Caused by styling_utils being moved into this folder

File: packages/core/test/bundling/core_all/bundle.golden_size_map_ivy.json [hunk 3]
```diff
@@ -177,48 +179,47 @@
...
+            "state.ts": 1311,
             "styling/": {
-              "size": 8202,
-              "bindings.ts": 2430,
-              "instructions.ts": 1821,
-              "map_based_bindings.ts": 1505,
-              "state.ts": 296,
-              "styling_debug.ts": 630,
-              "util.ts": 1520
+              "size": 5655,
```
Comment (kara) [inline packages/core/test/bundling/core_all/bundle.golden_size_map_ivy.json]: This decrease is offset by the styling files being moved elsewhere - util/ and instructions/ folders.

**Pull request closed without merging**